drivers/clock_control/nrf5_power_clock.c

```diff
@@ -17,6 +17,12 @@
 #include <drivers/clock_control/nrf5_clock_control.h>
 #endif
 
+#define HF_STAT_MASK (CLOCK_HFCLKSTAT_SRC_Xtal | CLOCK_HFCLKSTAT_STATE_Msk)
+
```

Comment:
what about creating local ERRATA_201 define as one may wonder why hacking nrf52.

```
#define ERRATA_201 IS_ENABLED(CONFIG_SOC_SERIES_NRF52X)
```

Than avoid using preprocessor in the code like:
```
if (ERRATA_201) {
...
}
```